tools/model_tools/README.md

```diff
@@ -29,17 +31,17 @@ future releases.
 
 ## Installation
 
-Model Downloader and other automation tools can be installed as a part of OpenVINO&trade; toolkit or from source.
+Model Downloader and other automation tools can be installed as part of the [openvino-dev](https://pypi.org/project/openvino-dev/) Python\* package or from source.
 Installation from source is as follows:
```

Comment:
```suggestion
To install the tools from source:
```

Reply:
Can we add more details about why customers may want to install from source and why they need to install the package first?